client/src/transmitter/index.ts

```diff
@@ -1,33 +1,75 @@
-import isIp from 'is-ip'
-
 import { copyToClipboard } from '../clipboard'
 import cast from './cast'
+import isLocalOrIp from './util/isLocalOrIp'
+
+const { body } = document
+
+const _video = document.getElementById('video') as HTMLVideoElement
 
-const isLocalHost = (hostname) => {
-  return hostname === 'localhost' || hostname.endsWith('.localhost')
+const _canvas = document.getElementById('canvas') as HTMLCanvasElement
+
+function resizeCanvas() {
+  const { innerWidth, innerHeight } = window
+
+  _canvas.width = innerWidth
+  _canvas.height = innerHeight
 }
 
-const { body } = document
+window.addEventListener('resize', () => {
+  resizeCanvas()
+})
 
-const video = document.getElementById('video')
-const playbackUrl = document.getElementById('playbackUrl')
+resizeCanvas()
 
-const record_container = document.getElementById('record-container')
-const record = document.getElementById('record')
+// @ts-ignore
```

Comment:
:watching-you:

Reply:
Support for `HTMLCanvasElement.captureStream` is so good; https://caniuse.com/?search=captureStream